config/sync/page_manager.page_variant.stanford_events_past-layout_builder-0.yml

```diff
@@ -47,27 +47,21 @@ variant_settings:
           uuid: b75e3d64-68b1-4eb0-afae-ef87817f748c
           region: left
           configuration:
-            id: 'menu_block:stanford-event-types'
-            label: 'Filter by event types'
-            provider: menu_block
-            label_display: visible
-            follow: false
-            follow_parent: child
-            level: 1
-            depth: 0
-            expand: false
-            parent: 'stanford-event-types:'
-            suggestion: stanford_event_types
-            label_type: block
-            label_link: false
+            id: 'inline_block:stanford_component_block'
+            label: Spacer
+            provider: layout_builder
+            label_display: '0'
+            view_mode: full
+            block_revision_id: 5
+            block_serialized: null
```

Comment:
What's this stuff? when I try to edit the layout for the past events page, i see this error:
`Error: Call to a member function getEntityTypeId() on null in Drupal\layout_builder\Plugin\Block\InlineBlock->build()`

Reply:
that's the error I tried to fix by rebuilding the page.  It was working correctly for me until I reverted the UUIDs

Reply:
I'd think the only change required in this config is replacing the view id name.

Reply:
that spacer block is in there to take the place of the sidebar menu in the layout. Otherwise, it pushes the results over to the right hand side.  I went over it with @ishahism because it seemed a little janky either way, and the concensus was to let the spacer on the right remain, since there's eventually going to be a filter block going back there anyway

Reply:
ok it sounds like you're mixing content and config then. It would be better to just change the layout settings to only take up 8 of the 12 columns instead of placing a spacer block. This can be done within the layout configuration settings when using the 1 column layout

Reply:
it's just a simple custom block created in layout builder, meant to put an empty div in the first column.  I'll remove it, since it appears to be breaking things.

Reply:
yeah a simple block is considered "content" in the eyes of Drupal.